src/python_testing/TC_CADMIN_1_5.py

```diff
@@ -46,15 +46,14 @@
 
 class TC_CADMIN_1_5(CADMINBaseTest):
 
-    async def commission_on_network(self, setup_code: int, discriminator: int, expected_error: int = 0):
+    async def commission_on_network(self, setup_code: int, discriminator: int, expected_chip_error: bool = False):
         # This is expected to error as steps 4 and 7 expects timeout issue or pase connection error to occur due to commissioning window being closed already
-        if expected_error == 50:
+        if expected_chip_error:
             with asserts.assert_raises(ChipStackError) as cm:
                 await self.th2.CommissionOnNetwork(
                     nodeId=self.dut_node_id, setupPinCode=setup_code,
                     filterType=ChipDeviceCtrl.DiscoveryFilterType.LONG_DISCRIMINATOR, filter=discriminator)
-            asserts.assert_true(int(cm.exception.code) == expected_error,
-                                'Unexpected error code returned from Commissioning Attempt')
+            asserts.assert_true(int(cm.exception.code) == 0, 'Unexpectedly Commissioning Attempt Passed')
```

Comment:
The `with asserts.assert_raises` asserts that there was exception, this code does nothing. Also you are using `cm` beyond context manager which does not look good.